src/platform_impl/ios/window.rs

```diff
@@ -271,8 +271,9 @@ impl Inner {
         unsafe { monitor::uiscreens() }
     }
 
-    pub fn primary_monitor(&self) -> MonitorHandle {
-        unsafe { monitor::main_uiscreen() }
+    pub fn primary_monitor(&self) -> Option<RootMonitorHandle> {
+        let handle = unsafe { monitor::main_uiscreen() };
+        Some(RootMonitorHandle { inner: handle })
```

Comment:
Would probably make sense to at least be consistent with the event_loop.rs variable naming here.